mkflame.sh

```diff
@@ -27,4 +27,4 @@ sh tslog.sh |
     perl flamechart.pl -flamechart -colors tslog --hash \
 	--title "`uname -r | cut -f 1-2 -d -` boot" \
 	--subtitle "$MS ms" \
-	--width $((MS / SCALE))
+	--width $(echo "$MS / $SCALE"|bc)
```

Comment:
Very minor style issue, can you add spaces around the `|` here?

Reply:
Sure! done